classes/mender-sdimg.bbclass

```diff
@@ -167,15 +167,18 @@ EOF
     # Call WIC
     IMAGE_CMD_wic
 
+    wicimgname="${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.wic"
```

Comment:
Yeah, I'm building with master only. We'll need to cherry pick 9793035a68434a0da1b3f4633a0de3a8d9be3a1a for krogoth manually.